weekly_digest/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
If we think the admin will need me able to access the subscriptions via the Django admin we need to register the WeeklyDigestSubscription model here.